src/js/components/TextInput/doc.js

```diff
@@ -92,3 +92,66 @@ suggestions and instead rely on the user to type more.`,
 
   return DocumentedTextInput;
 };
+
+export const themeDoc = {
+  'global.colors.placeholder': {
+    description: 'The placeholder color used for TextInput.',
+    type: 'string',
+    defaultValue: '#AAAAAA',
+  },
+  'global.control.border.width': {
+    description: 'The border width.',
+    type: 'string',
+    defaultValue: '1px',
+  },
+  'global.input.padding': {
+    description: 'The padding of the text.',
+    type: 'string',
+    defaultValue: '12px',
+  },
+  'global.input.weight': {
+    description: 'The font weight of the label.',
+    type: 'number',
+    defaultValue: 600,
+  },
+  'select.step': {
+    description: 'How many items to render at a time.',
```

Comment:
I think we should make this description a bit more oriented toward TextInput by saying "How many suggestions to render at a time."